packages/web/src/scss/components/TextArea/README.md

```diff
@@ -146,6 +146,23 @@ JS interaction class when controlled by JavaScript:
 </div>
 ```
 
+### V1 Disabled State Style
+
+To enable a new disabled style, add the feature class `spirit-v1-box-field-disabled` to a parent element.
+The preferred one is the `body` element because this way it will affect all TextArea components everywhere.
```

Comment:
`preferred` - Is there another way?

Reply:
You can place the class to any parent element and it will work. But we prefer `body` as it is in one place with other feature classes.